test/multiverse/suites/roda/ignorer_test.rb

```diff
@@ -0,0 +1,203 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require_relative '../../../../lib/new_relic/agent/instrumentation/roda/instrumentation'
+require_relative '../../../../lib/new_relic/agent/instrumentation/roda/roda_transaction_namer'
+require_relative '../../../../lib/new_relic/agent/instrumentation/roda/ignorer'
+
+JS_AGENT_LOADER = 'JS_AGENT_LOADER'
+
+def assert_enduser_ignored(response)
+  refute_match(/#{JS_AGENT_LOADER}/o, response.body)
+end
+
+def refute_enduser_ignored(response)
+  assert_match(/#{JS_AGENT_LOADER}/o, response.body)
+end
+
+def fake_html_for_browser_timing_header
+  '<html><head><title></title></head><body></body></html>'
+end
```

Comment:
Would it work for you to move these methods into the class that calls them? Optionally, you could make them private.

Reply:
Made them private since more than one class uses them 0a4c1404dbf04f2fa313cd07e3dfca27ef63142f

Reply:
This looks better 😎 3b755c1cee3e25f8f02002427f0194d08d768606